api/net/Areas/Services/Controllers/ContentController.cs

```diff
@@ -344,22 +344,25 @@ public IActionResult DownloadFile(long id)
     /// <returns></returns>
     [HttpGet("{id}/image")]
     [ProducesResponseType(typeof(string), (int)HttpStatusCode.OK)]
-    [ProducesResponseType((int)HttpStatusCode.BadRequest)]
+    [ProducesResponseType((int)HttpStatusCode.NoContent)]
```

Comment:
I just made this change and we want to keep the BadRequest.  Handle the bad request in the Reporting Service.

Reply:
Updated, thanks